lambda/models/state_machine/create_model.py

```diff
@@ -97,14 +99,61 @@ def handle_poll_docker_image_available(event: Dict[str, Any], context: Any) -> D
 def handle_start_create_stack(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Start model infrastructure creation."""
     output_dict = deepcopy(event)
+    request = CreateModelRequest.validate(event)
+
+    prepared_event = {}
+
+    def camelize_object(o):
+        o2 = {}
+        for k in o:
+            fixed_k = k[:1].lower() + k[1:]
+            if ( isinstance(o[k], dict) ):
+                o2[fixed_k] = camelize_object(o[k])
+            else:
+                o2[fixed_k] = o[k]
+        return o2
+            
+    prepared_event = camelize_object(event)
+    prepared_event["containerConfig"]["image"] = {
+        "repositoryArn": os.environ["ECR_REPOSITORY_ARN"],
+        "tag": event["image_info"]["image_tag"],
+        "type": "ecr"
+    }
+
+    response = lambdaClient.invoke(
+        FunctionName=os.environ['ECS_MODEL_DEPLOYER_FN_ARN'],
+        Payload=json.dumps({"modelConfig": prepared_event}),
+    )
+
+    payload = response["Payload"].read()
+    payload = json.loads(payload)
+    stack_name = payload.get('stackName')
+    output_dict["stack_name"] = stack_name
```

Comment:
could we change this to a stack arn instead? the whole arn with the trailing UUID will help us identify the stack, even if it is ultimately rolled back and deleted before we're done monitoring it, otherwise if it gets deleted and we don't have its uuid, we may not be able to track it after a while

example for a stack whose name is `my-model-stack`:

```
arn:aws:cloudformation:us-east-1:000000000000:stack/my-model-stack/dafea8c0-a96b-11ec-af1e-1285d9cd23a9
```

Reply:
yeah, worst case I can just do a `describeStacks` by name right after the lambda returns. Not sure if there's going to be a way to get the stack arn out of cdk before it finishes deploying

Reply:
yea that works- i'm good with describe to get the arn. the arn will be most consistent to use in the rest of the state machine.